test/nn/test_convolution.py

```diff
@@ -506,10 +506,11 @@ def test_Conv2d_groups_nobias(self):
             output2 = m2(i2)
             output2.backward(grad_output[:, 2:].contiguous())
 
-            self.assertEqual(output, torch.cat([output1, output2], 1))
+            self.assertEqual(output, torch.cat([output1, output2], 1),
+                             atol=dtype2prec_DONTUSE[dtype], rtol=0)
             self.assertEqual(i.grad.data,
                              torch.cat([i1.grad.data, i2.grad.data], 1),
-                             atol=dtype2prec_DONTUSE[dtype], rtol=0)
+                             atol=1e-1 if dtype == torch.half else dtype2prec_DONTUSE[dtype], rtol=0)
```

Comment:
Why is the default one not good?

Reply:
IIRC on Navi 3x, due to the usage of WMMA instructions the tolerance for 16-bit data types are getting worse so the tolerance was raised even further.

P.S. according to the comment of dtype2prec_DONTUSE:
> # Using @toleranceOverride specific to your test is the recommended way
> # of doing this. These are just some values that worked for test_nn.

Reply:
@albanD We'll follow-up with our internal teams to understand a bit better why the tolerance requirements are needed for both MI2xx as well as Navi3x. Taking the PR back to Draft for now.